test/e2e/run_volume_test.go

```diff
@@ -554,7 +542,7 @@ VOLUME /test/`, ALPINE)
 			Skip("Overlay mounts not supported when running in a container")
 		}
 		if rootless.IsRootless() {
-			if _, err := exec.LookPath("fuse_overlay"); err != nil {
+			if _, err := exec.LookPath("fuse-overlayfs"); err != nil {
```

Comment:
This was a typo. Tests were never running rootless, because there is no such binary as `fuse_overlay`.

FIXME: is `fuse-overlayfs` needed even in the new world where overlay is supported in the kernel? Does this conditional need to change?

Reply:
Yes I think this should be changed, @flouthoc @giuseppe WDYT?

Reply:
I agree, we should run the tests no matter what.

Reply:
Yes i remember changing it for some new tests https://github.com/containers/podman/pull/11650/files but it should be done across the code.